common.mk

```diff
@@ -73,15 +73,15 @@ linux-common: linux-defconfig
 linux-defconfig-clean-common:
 	@rm -f $(LINUX_PATH)/.config
 
-LINUX_CLEAN_COMMON_FLAGS ?= $(LINUX_COMMON_FLAGS)
+#LINUX_CLEAN_COMMON_FLAGS ?= UNCOMMENT_AND_DEFINE_AS_NECESSARY
```

Comment:
@pascal-brand-st-dev Do you prefer that just for LINUX_CLEAN_COMMON_FLAGS or all the *CLEAN_COMMON_FLAGS?

@jbech-linaro Yes, ARCH is propagated here from the <platform>.mk even if LINUX_CLEAN_COMMON_FLAGS is commented out here, so why do we need to define CROSS_COMPILE for cleaning? Nothing is built.